bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/targetComponent/EdCompTargetList.java

```diff
@@ -1375,17 +1376,20 @@ else if (contents instanceof TableGroupSelection) {
         if (isMember(env, target)) return;
         env = env.removeTarget(target);
 
-        final SPTarget base = env.getArbitraryTargetFromAsterism();
+        final SPSkyObject base = env.getSlewPositionObjectFromAsterism();
+        if (!(base instanceof SPTarget))
+            return;
+        final SPTarget baseTarget = (SPTarget) base;
 
         final GuideEnvironment   genv = env.getGuideEnvironment();
-        final ImList<UserTarget> user = env.getUserTargets().append(new UserTarget(UserTarget.Type.other, base));
+        final ImList<UserTarget> user = env.getUserTargets().append(new UserTarget(UserTarget.Type.other, baseTarget));
 
-        final TargetEnvironment newEnv = TargetEnvironment.create(target, genv, user);
+        final TargetEnvironment newEnv = TargetEnvironment.create(baseTarget, genv, user);
```

Comment:
How does this work when the `Asterism` is a `GhostAsterism` since this `create` method will return an environment with a `Single` target asterism?

Reply:
Very good point. That slipped by me entirely. I think that that would result in an incorrect state. I'm not really sure about this morphing, given the dependency of GHOST base on the target(s). Maybe we should just disallow it for GHOST?

Reply:
Yeah it seems like this would require a lot more thought for asterisms in general.  It's also a pretty dubious feature, imho.  Perhaps we can hide the target type change combo altogether for now unless the asterism is `Single` target.

Reply:
Sure. I think that sounds like the best option.